local/blockchain.go

```diff
@@ -683,6 +677,7 @@ func (ln *localNetwork) waitSubnetValidators(
 				nodeID := node.GetNodeID()
 				if isValidator := subnetValidators.Contains(nodeID); !isValidator {
 					ready = false
+					break subnetLoop
```

Comment:
I'm assuming we do want to break here?

Reply:
we want to wait for waitForValidatorsPullFrequency

Reply:
With this change, we will still do that. Without this `break` we will do a bunch of useless work -- namely we will do the logic inside this loop:

```go
for _, subnetID := range subnetIDs {
```

even though the logic inside that loop doesn't matter because `ready` is already false.